functests/4_latency/latency.go

```diff
@@ -56,8 +55,15 @@ var _ = Describe("[performance] Latency Test", func() {
 	var workerRTNode *corev1.Node
 	var profile *performancev2.PerformanceProfile
 	var latencyTestPod *corev1.Pod
+	var err error
+	latencyTestDelay, err = getLatencyTestDelay()
```

Comment:
If I understood correctly, this PR intended to check the environment variables each time for each tool.
As @MarSik mentioned, storing the values here, won't achieve that, because the ```Describe()``` is common to all three test cases.
You can call theses ```get*()``` functions inside each ```Context()``` or set the global variables (```latencyTestDelay```, ```latencyTestRuntime```, etc.)  in the ```BeforeEach()```

Reply:
fixed, thanks.